src/python/pants/backend/python/goals/pytest_runner_integration_test.py

```diff
@@ -51,6 +61,7 @@ def rule_runner() -> RuleRunner:
 BAD_SOURCE = FileContent(f"{PACKAGE}/test_bad.py", b"def test():\n  assert False\n")
 PY3_ONLY_SOURCE = FileContent(f"{PACKAGE}/test_py3.py", b"def test() -> None:\n  pass\n")
 LIBRARY_SOURCE = FileContent(f"{PACKAGE}/library.py", b"def add_two(x):\n  return x + 2\n")
+BINARY_SOURCE = FileContent(f"{PACKAGE}/say_hello.py", b"print('Hello, test!')")
```

Comment:
Nit: better to inline this, as it's only used by one very specific test.

Reply:
I think I prefer the uniformity, TBH.